### PR TITLE
Add doc for installing CA clone with secure DS

### PR DESCRIPTION
**Pull request by PsOverflow**
This PR adds a Doc for installing a Clone CA instance with secure
DS connection.

### CODE REVIEW

**Comment by edewata** [comment]
Preview: https://github.com/dogtagpki/pki/blob/Add-doc-for-installing-CA-clone-with-secure-DS/docs/installation/ca/Installing_CA_Clone_with_Secure_Database_Connection.md

**Review by edewata — COMMENTED** [review]
Thanks for creating the doc! Please take a look at my suggestions below.

One more thing, right now we have ~80 CI tests, but this PR ran ~160 CI tests. Apparently this is because you pushed your branch directly to [dogtagpki/pki](https://github.com/dogtagpki/pki), so together with the PR it ran the CI tests twice. In the future please push your branch into your fork (i.e. [PsOverflow/pki](https://github.com/PsOverflow/pki)), then you can create a PR against [dogtagpki/pki](https://github.com/dogtagpki/pki).

File: docs/installation/ca/Installing_CA_Clone_with_Secure_Database_Connection.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+Before beginning with the installation, please ensure that you have configured the directory
+server as described in [DS Installation](https://github.com/dogtagpki/pki/wiki/DS-Installation).
+
+Then on existing system, export the DS signing certificate into `ds_signing.p12` and copy the certificate into clone system with the following command:
+
+```
+pki -d /etc/dirsrv/slapd-localhost \
+-C /etc/dirsrv/slapd-localhost/pwdfile.txt \
+pkcs12-export --pkcs12-file ds_signing.p12 \
+--pkcs12-password Secret.123 Self-Signed-CA
```
[the remark this answers is not in the document]
Comment (PsOverflow) [reply docs/installation/ca/Installing_CA_Clone_with_Secure_Database_Connection.md:21]: @edewata @ladycfu  Sorry. By mistake directly created this PR for this Doc.
I am closing this PR.
Fixed the suggested changes at https://github.com/dogtagpki/pki/pull/3486

**Comment by PsOverflow** [comment]
Added suggested fix at https://github.com/dogtagpki/pki/pull/3486.
Closing this PR.

**Pull request closed without merging**